flow-sample-basic/src/main/java/flow/sample/basic/Screens.java

```diff
@@ -0,0 +1,26 @@
+package flow.sample.basic;
+
+import android.content.Context;
+
+final class Screens {
+  private static final String SCREEN = "SERVICE_SCREEN";
+
+  public static <T> T getScreen(Context context) {
+    //noinspection unchecked,ResourceType
+    return (T) context.getSystemService(SCREEN);
+  }
+
+  static final class ContextWrapper extends android.content.ContextWrapper {
```

Comment:
Usual boilerplate:

```
  @Override public Object getSystemService(String name) {
    if (LAYOUT_INFLATER_SERVICE.equals(name)) {
      if (inflater == null) {
        inflater = LayoutInflater.from(getBaseContext()).cloneInContext(this);
      }
      return inflater;
    }
    return scope.hasService(name) ? scope.getService(name) : super.getSystemService(name);
  }
```

Get it from the Flow. Have the View implement an interface with a setScreen() method. Whatever.

Not a blocker, just a conversation.